erpnext/manufacturing/doctype/bom_update_log/bom_updation_utils.py

```diff
@@ -157,12 +157,21 @@ def _all_children_are_processed(parent_bom):
 def get_leaf_boms() -> List[str]:
 	"Get BOMs that have no dependencies."
 
-	return frappe.db.sql_list(
-		"""select name from `tabBOM` bom
-		where docstatus=1 and is_active=1
-			and not exists(select bom_no from `tabBOM Item`
-				where parent=bom.name and bom_no !='')"""
-	)
+	bom = frappe.qb.DocType("BOM")
+	bom_item = frappe.qb.DocType("BOM Item")
+
+	boms = (
+		frappe.qb.from_(bom)
+		.left_join(bom_item)
+		.on((bom.name == bom_item.parent) & (bom_item.bom_no != ""))
+		.select(bom.name)
+		.where((bom.docstatus == 1) & (bom.is_active == 1) & (bom_item.bom_no.isnull()))
+		.distinct()
+	).run(as_list=True)
+
+	boms = [d[0] for d in boms]
+
```

Comment:
```suggestion
	).run(pluck=True)
```

This is equivalent iirc.

Reply:
IDK if `pluck=True` going to work in v13?

Reply:
Ah, idk for sure. Can switch in port. v13 has short life left anyway.